### PR TITLE
Filters

### PR DESCRIPTION
**Pull request by WilliamLQin**

### CODE REVIEW

File: website/components/SearchBar/SearchBar.tsx
Comment (a28dong) [inline website/components/SearchBar/SearchBar.tsx:1]: This is free to remove right?
@bryanling1

File: website/lib/extension/jobFilters.ts [hunk 1]
```diff
@@ -0,0 +1,88 @@
+export enum JobFilters {
+  appDocFilter = "Required Documents",
+  durationFilter = "Duration",
+  specialReqFilter = "Special Requirements",
+}
+
+export enum DurationFilterTags {
+  fourMonth = "4 month",
```
Comment (a28dong) [inline website/lib/extension/jobFilters.ts:8]: Why lowercase?
Do we want the display of this to be lowercase in the chip?

**Review by a28dong — APPROVED** [review]
lgtm

File: website/lib/extension/jobFilters.ts (hunk 1 above)
Comment (WilliamLQin) [reply website/lib/extension/jobFilters.ts:8]: it looks weird upper case imo

**Pull request merged**